src/core/composer/qgscomposerpolyline.cpp

```diff
@@ -136,7 +136,7 @@ void QgsComposerPolyline::setPolylineStyleSymbol( QgsLineSymbolV2* symbol )
   emit frameChanged();
 }
 
-void QgsComposerPolyline::_writeXMLStyle( QDomDocument &doc, QDomElement &elmt ) const
+void QgsComposerPolyline::_writeXmlStyle( QDomDocument &doc, QDomElement &elmt ) const
```

Comment:
Do we keep the `_`-prefix?
No need to do it in this PR but I think it's something from python (were people are desperately missing private-sections ;) )